packages/dd-trace/src/config.js

```diff
@@ -194,6 +202,7 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
     this.tracing = !isFalse(DD_TRACING_ENABLED)
     this.debug = isTrue(DD_TRACE_DEBUG)
     this.logInjection = isTrue(DD_LOGS_INJECTION)
+    this.qsObfuscator = DD_OBFUSCATION_QUERY_STRING_REGEXP
```

Comment:
How can a user _add_ their own custom regular expression without disabling ours?

Reply:
They copy our regex and add their own to it

Reply:
But then they would need to update their regex when ours change. Not all users will want to override the default completely, they might just want to add additional rules specific to them.

Reply:
That can be a later improvement, for now this is not part of the spec.